src/situations/securite/styles/situation.scss

```diff
@@ -21,8 +21,43 @@
   position: absolute;
   top: 3%;
   right: 3%;
+  display: flex;
+  align-items: center;
   border-radius: 1rem;
   background-color: $blanc;
   padding: 1rem;
   color: $couleur-sombre;
 }
+
+@keyframes agrandit-puis-retrecit {
+  50% {
+    transform: scale(1.5);
+  }
+
+  to {
+    transform: scale(1);
+  }
+}
+
+.panneau-danger {
+  margin-right: .5rem;
+
+  &:first-of-type {
```

Comment:
avec un `display:grid` on n'aurait pas besoins de jongler avec les `first-of-type` et `last-of-type` ;-)